cg/store/database.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session, scoped_session, sessionmaker
+
+from cg.exc import CgError
+from cg.store.models import Model
+
+SESSION: Optional[Session] = None
+ENGINE = None
+
+
+def initialise_database(db_uri: str):
+    global SESSION, ENGINE
+    ENGINE = create_engine(db_uri, pool_pre_ping=True)
+    session_factory = sessionmaker(ENGINE)
```

Comment:
would this not throw errors if the connection cannot be established?

Reply:
Not sure, cannot find any docs for it 🤔 But I would guess so. It has the same behaviour as before at least.